index.css

```diff
@@ -0,0 +1,52 @@
+html, body {
+    height: 100%;
+}
+
+body {
+    background-color: lavender;
+    font-family: "Courier New", Courier, monospace;
+}
+
+#middle {
+    width: 70%;
+    margin: auto;
+}
+
+#bottom {
+    width: 60%;
+    margin: auto;
+}
+
+#banner {
+    background-color: skyblue;
+    width: 70%;
+    height: 150px;
+    margin: auto;
+    margin-bottom: 30px;
+    padding-left: 10px;
+    padding-right: 10px;
+    display: grid;
```

Comment:
`display: flex` would have been easier here. I would prevent you from having to specify the columns that the header and image must span. You can talk to @lakerfan1994 (Chuck) to see how he approached this and what to do for our website to be more responsive.